pages/home_page.py

```diff
@@ -24,6 +26,13 @@ def __init__(self, testsetup, product=None):
 
         if product is None:
             self.selenium.get(self.base_url)
+        self.wait_for_page_to_load()
+
+    def wait_for_page_to_load(self):
+        Wait(self.selenium, self.timeout).until(
+            lambda s: 'loading' not in s.find_element(
+                *self._graph_locator).get_attribute('class'))
+        return self
```

Comment:
It returns the page object. It's not essential or even used in this patch, I've just been using it in recent projects. It allows for executing like like:

```python
page = MyPage(self.testsetup).wait_for_page_to_load()
```

Instead of:

```python
page = MyPage(self.testsetup)
page.wait_for_page_to_load()
```